src/components-examples/material/dialog/dialog-from-menu/dialog-from-menu-example.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, ViewChild} from '@angular/core';
+import {MatDialog} from '@angular/material/dialog';
+import {MatMenuTrigger} from '@angular/material/menu';
+/**
+ * @title Dialog launched from a menu
+ */
+@Component({
+  selector: 'dialog-from-menu-example',
+  templateUrl: 'dialog-from-menu-example.html',
+})
+export class DialogFromMenuExample {
+  @ViewChild('menuTrigger') menuTrigger: MatMenuTrigger;
+
+  constructor(public dialog: MatDialog) {}
+
+  openDialog() {
+    // #docregion focus-restoration
+    const dialogRef = this.dialog.open(DialogFromMenuExampleDialog, {restoreFocus: false});
+
+    dialogRef.afterClosed().subscribe(() => this.menuTrigger.focus());
```

Comment:
Can you add a comment here like
```ts
// Manually restore focus to the menu trigger since the element that
// opens the dialog won't be in the DOM any more when the dialog closes.
```